Mathlib/RingTheory/Polynomial/Quotient.lean

```diff
@@ -70,6 +70,14 @@ noncomputable def quotientSpanCXSubCXSubCAlgEquiv {x : R} {y : R[X]} :
       (Ideal.Quotient.algebra R) _ :=
 ((quotientSpanCXSubCAlgEquiv (X - C x) y).restrictScalars R).trans <| quotientSpanXSubCAlgEquiv x
 
+lemma modByMonic_eq_zero_iff_quotient_eq_zero (p q : R[X]) (hq : q.Monic) :
+    p %ₘ q = 0 ↔ (p : R[X] ⧸ Ideal.span {q}) = 0 := by
+  rw [dvd_iff_modByMonic_eq_zero hq, Ideal.Quotient.eq_zero_iff_dvd]
+
+@[simp]
+lemma quotient_singleton_eq (p : R[X]) : Ideal.Quotient.mk (Ideal.span {p}) p = 0 := by
+  rw [Ideal.Quotient.eq_zero_iff_dvd]
```

Comment:
This lemma has nothing to do with polynomials

Reply:
Thanks, moved this to right after the lemma it uses